.workflow-version

```diff
@@ -0,0 +1 @@
+$workflow_version='1.0.0.alpha1'
```

Comment:
constants in Ruby tend to be `ALL_CAPS_SNAKE_CASE`

in the current implementation the `$` makes me think this is an environment variable or a shell script variable, is there a reason for the `$`?

Reply:
No reason, will switch to  all caps, snake case.

Reply:
switching it will introduce issues with `load` mentioned above, so if we're sticking with `load` then we shouldn't use the all-caps version (Ruby actually treats them as constants)